ingress/controllers/gce/controller/utils.go

```diff
@@ -40,6 +40,8 @@ import (
 const (
 	allowHTTPKey    = "kubernetes.io/ingress.allow-http"
 	staticIPNameKey = "kubernetes.io/ingress.global-static-ip-name"
+	ingressClassKey = "kubernetes.io/ingress.class"
```

Comment:
this is a new dedicated annotation right?
who is going to set this? Customer? kubectl?